src/lms-harmonizer/tests_integration_sql/conftest.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from typing import Iterable
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine, Connection, Transaction
+from tests_integration_sql.migrator_helper_mssql import migrate_lms_user_and_edfi_student
+
+
+def _new_mssql_engine() -> Engine:
+    """
+    Assumes existence of local SQLServer DB named 'test_harmonizer_lms_toolkit'
+    """
+    return create_engine(
+        "mssql+pyodbc://localhost,1433/test_harmonizer_lms_toolkit?driver=ODBC+Driver+17+for+SQL+Server?Trusted_Connection=yes",
```

Comment:
Can we get a mechanism to allow this to run on a different server? Maybe just try to read some environment variables, and default to "localhost", 1433, and "test_harmonizer_lms_toolkit" if those env vars aren't set.

This way we can run against SQL Server in a container or on another server.

Oh, to run in a container we need to specify the username and password, since domain authentication won't work. If needed we can make this request into a separate ticket.

Reply:
A good idea. Probably best to put in a separate ticket.

Reply:
https://tracker.ed-fi.org/browse/LMS-327